### ISSUE TITLE
Doesn't work with explicit transitive deps

### ISSUE THREAD

**Issue by mimoo** [janestreet/ppx_sexp_conv#33]
Hey!

When setting [implicit_transitive_deps](https://dune.readthedocs.io/en/stable/dune-files.html#implicit-transitive-deps) to `false` in a `dune-project` it complains if I use ppx_sexp_conv:

```
Error: The module Ppx_sexp_conv_lib.Sexp is an alias for module Sexplib0.Sexp, which is missing
```

**Comment by aalekseyev** [comment]
When using `implicit_transitive_deps false`, you should be prepared to have to add random things to your jbuild files. In this case I think you have to add `sexplib0`.

Closing since this is expected. Feel free to reopen if you think we can/should do something on our end about it.

**Comment by mimoo** [comment]
That's not really what the documentation of implicit_transitive_deps implies. I'm relatively new to OCaml so I'm not sure if this is the solution, but I've seen other ppx libraries mentioning "runtime" libraries that you have to include in addition to the ppx.

**Comment by aalekseyev** [comment]
Oh, I see. Since this is a ppx rewriter, we do have a mechanism to make `implicit_transitive_deps false` work with no annotation. 
Since it's a simple change, I'll make it, but I do think it's weird that `implicit_transitive_deps false` creates this compatibility burden for the individual sexp rewriters written and tested against `implicit_transitive_deps true`.

**Comment by aalekseyev** [comment]
This change landed in 5ae3e96d6703802a02ea7d190a33caec0376e95d.
